bls-signatures-ffi/src/lib.rs

```diff
@@ -154,13 +155,39 @@ pub unsafe extern "C" fn verify(
 }
 
 /// Generate a new private key
+#[no_mangle]
+pub unsafe extern "C" fn private_key_generate() -> *mut responses::PrivateKeyGenerateResponse {
+    let rng = &mut OsRng::new().expect("not enough randomness");
+
+    let mut raw_private_key: [u8; PRIVATE_KEY_BYTES] = [0; PRIVATE_KEY_BYTES];
+    PrivateKey::generate(rng)
+        .write_bytes(&mut raw_private_key.as_mut())
+        .expect("preallocated");
+
+    let response = responses::PrivateKeyGenerateResponse {
+        private_key: raw_private_key,
+    };
+
+    Box::into_raw(Box::new(response))
+}
+
+/// Generate a new private key with seed
 ///
```

Comment:
lets add a warning that this should be used only for testing or with very secure seeds

Reply:
Done.